### ISSUE TITLE
Does it support the smart Humidifier?

### ISSUE THREAD

**Issue by EpicMount** [LaggAt/hacs-govee#104]
Is there a list of devices that are supported by this integration?  I have a bunch of their lights in my set up that are working fine after I was able to tweak the API refresh rate, but I just purchased and set up 2 of their smart humidifiers which I'd like to control with a simple on/off command however I do not see them being picked up by this integration. The specific model I am referring to is the H7141.

Thanks!

**Comment by Jay1175** [comment]
Would also be interested, as I'd like to control my govee fan via temp/humidity sensors

**Comment by rmcginty** [comment]
Same here. Tower fan.

**Comment by rmcginty** [comment]
Same here. Tower fan.

**Comment by ColinKlayer** [comment]
I've got the H7160 humidifier that I'd love to get connected. I've got it in HomeKit by way of Homebridge but it's not showing up through HomeKit Controller.

**Comment by alex4108** [comment]
Hmm, bump.  Would be interested in implementing

**Comment by fjrichman** [comment]
New API supports both Lights and Appliances: https://developer.govee.com/

**Comment by optsysonl** [comment]
Yes, a separate issue has been raised for the new API support; #138
